src/Symfony/Bridge/Doctrine/DataCollector/DoctrineDataCollector.php

```diff
@@ -162,10 +198,14 @@ private function sanitizeQuery(string $connectionName, array $query): array
                 }
             }
 
-            list($query['params'][$j], $explainable) = $this->sanitizeParam($param);
+            list($query['params'][$j], $explainable, $runnable) = $this->sanitizeParam($param, $e);
             if (!$explainable) {
                 $query['explainable'] = false;
             }
+
+            if (!$runnable) {
+                $query['runnable'] = false;
```

Comment:
shouldn't it also be set to false if we cannot convert to database value, as done for explainable ?

Reply:
OK, it looks like you already handle it in `sanitizeParam`. But then, assigning `explainable` in these catch clauses might be cleaned as well.

Reply:
Good catch, I cleaned them.